README.md

```diff
@@ -318,8 +321,7 @@ bundle exec cap india:staging deploy:rake task=db:seed
 
 ### Deployment Resources
 
-* [Deployment instructions for Simple Server](doc/deployment.md)
```

Comment:
I've kept this file in `wiki` but removed it from this link. I'm not certain if that `deployment.md` doc is still valuable.

Reply:
A deployment doc is in general valuable, but this one is deprecated. More up-to-date docs are in the readme: https://github.com/simpledotorg/simple-server#deployment

Wanna link to that instead?

Reply:
Ah, in which case I'll just delete this file for now. The README one is good enough as is.